test/functional/lua/hl_spec.lua

```diff
@@ -104,6 +104,30 @@ describe('vim.hl.range', function()
                                                                   |
     ]])
   end)
+
+  it('correctly removes highlight after timeout time', function ()
```

Comment:
"correctly" is never needed in test names.
```suggestion
  it('removes highlight after given `timeout`', function ()
```